src/selectedtests/app/test_mapping_work_item.py

```diff
@@ -0,0 +1,120 @@
+"""Model of Evergreen TestMapping that needs to be analyzed."""
```

Comment:
This file is following the conventions of https://github.com/10gen/dag-metrics/blob/master/src/dagmetrics/nr_metrics/version_work_item.py